src/main/java/wolox/training/models/Book.java

```diff
@@ -27,6 +31,7 @@
     @Column(nullable = false)
     private String image;
 
+    @ApiModelProperty(notes = "Book's title")
```

Comment:
can you are add this format "@ApiModelProperty(notes = "Book's tittle", required = true)", so you indicate that this field is required